src/features/entities/person-details.tsx

```diff
@@ -22,31 +23,40 @@ export function PersonDetails(props: PersonDetailsProps): JSX.Element {
 
   const hasRelations = person.relations != null && person.relations.length > 0;
 
+  const hasBiographies = person.biographies != null && person.biographies.length > 0;
+
   const hasMedia = person.media != null && person.media.length > 0;
 
   return (
     <div className="mx-auto grid w-full max-w-6xl content-start gap-4 px-8 py-12">
       <EntityTitle kind={person.kind} label={person.label} />
-      <EntityAlternativeLabels labels={alternativeLabels} />
-      <EntityLinkedIds links={person.linkedIds} />
-      <EntityDescription description={person.description} />
-      <PersonMetadataList gender={person.gender} occupations={person.occupations} />
-      {hasRelations ? <EntityRelations relations={person.relations} /> : null}
-      {hasRelations ? (
-        <NetworkComponent
-          visualization={{
-            id: `ego-network-${person.id}`,
-            type: 'ego-network',
-            name: `ego-network-${person.id}`,
-            entityIds: [person.id],
-            targetEntityIds: [],
-            eventIds: [],
-          }}
-          width={600}
-          height={600}
-        />
-      ) : null}
-      {hasMedia ? <MediaViewer mediaResourceIds={person.media!} /> : null}
+      <div className="grid grid-cols-2 gap-4">
```

Comment:
gap-4 seems a bit small to me. gap-16 separates the columns better (at least on my screen)

Reply:
I pushed this change, revert if doesn't look good